sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -0,0 +1,460 @@
+| Text Blocklist Management APIs|The default AI classifiers are sufficient for most content safety needs. However, you might need to screen for terms that are specific to your use case. You can create blocklists of terms to use with the Text API.
+
+### Harm categories
+Content Safety recognizes four distinct categories of objectionable content.
+|Category	|Description  |
+|---------|---------|
+|Hate	|Hate refers to any content that attacks or uses pejorative or discriminatory language in reference to a person or identity group based on certain differentiating attributes of that group. This includes but is not limited to race, ethnicity, nationality, gender identity and expression, sexual orientation, religion, immigration status, ability status, personal appearance, and body size.|
+|Sexual	|Sexual describes content related to anatomical organs and genitals, romantic relationships, acts portrayed in erotic or affectionate terms, pregnancy, physical sexual acts—including those acts portrayed as an assault or a forced sexual violent act against one’s will—, prostitution, pornography, and abuse.|
+|Violence	|Violence describes content related to physical actions intended to hurt, injure, damage, or kill someone or something. It also includes weapons, guns and related entities, such as manufacturers, associations, legislation, and similar.|
+|Self-harm	|Self-harm describes content related to physical actions intended to purposely hurt, injure, or damage one’s body or kill oneself.|
+Classification can be multi-labeled. For example, when a text sample goes through the text moderation model, it could be classified as both Sexual content and Violence.
+
+### Severity levels
+Every harm category the service applies also comes with a severity level rating. The severity level is meant to indicate the severity of the consequences of showing the flagged content.
+|Severity	|Label	|
+|---------|---------|
+|0	|Safe|
+|2	|Low|
+|4	|Medium|
+|6	|High|
+A severity of 0 or "Safe" indicates a negative result: no objectionable content was detected in that category.
+
+### Text blocklist management
+Following operations are supported to manage your text blocklist:
+- Create or modify a blocklist
+- List all blocklists
+- Get a blocklist by blocklistName
+- Add blockItems to a blocklist
+- Remove blockItems from a blocklist
+- List all blockItems in a blocklist by blocklistName
+- Get a blockItem in a blocklist by blockItemId and blocklistName
+- Delete a blocklist and all of its blockItems
+
+You can set the blocklists you want to use when analyze text, then you can get blocklist match result from returned response.
+
+## Examples
+
+The following section provides several code snippets covering some of the most common Content Safety service tasks, including:
+
+- [Analyze text](#analyze-text)
+- [Analyze image](#analyze-image)
+- [Manage text blocklist](#manage-text-blocklist)
+
+### Analyze text
+
+```python
```

Comment:
This should just take a minute to add. You can decorate each of your snippets with the following so that if you update your samples you can simply run the tool to keep them in sync:
```
<!-- SNIPPET:sample_authentication.create_da_client_with_aad -->

...snippet content goes here...

<!-- END SNIPPET -->
```
Here is a link with more info: https://github.com/Azure/azure-sdk-for-python/blob/06f4a7db037fb01d0bc05ecb3f41666092999683/doc/dev/code_snippets.md#refer-samples

You can do this in a separate PR if you want, just open a tracking issue

Reply:
Updated with code snippet.